hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -508,3 +528,11 @@ def stop_at_end_of_cycle(self, err_bits: int) -> None:
 
     def invalidate_imem(self) -> None:
         self._time_to_imem_invalidation = 2
+
+    def start_wiping(self) -> None:
+        self.ext_regs.increment_insn_cnt()
+        self.wiping_cnt = 0
+        if self._err_bits:
+            # Abort all pending changes, including changes to external
+            # registers.
+            self._abort()
```

Comment:
How does ERR_BITS get set correctly once wiping is done? I think this might need a bit more thought. We probably just need to change the code that writes to ERR_BITS to do so at a different time?